### PR TITLE
OSDOCS-6804 Z-stream RNs for 4.12.24

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.12

Issue:
[OSDOCS-6804](https://issues.redhat.com/browse/OSDOCS-6804)

Link to docs preview:
[Preview](https://62158--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-24)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -2782,6 +2782,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |General Availability
 
+|NUMA Resources Operator
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:2785]: I think the best spot for this is table 22 `Scalability and performance`

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -3811,4 +3816,45 @@ $ oc adm release info 4.12.23 --pullspecs
 [id="ocp-4-12-23-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-24"]
+=== RHBA-2023:3977 - {product-title} 4.12.24 bug fix update and security update
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:3822]:
```suggestion
=== RHBA-2023:3977 - {product-title} 4.12.24 bug fix and security update
```

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:2785]:
```suggestion
|NUMA-aware scheduling with NUMA Resources Operator
```

**Review by JoeAldinger — COMMENTED** [review]
Small typo and a suggestion for placement in the tables.

**Comment by JoeAldinger** [comment]
cc @rohennes making a note that https://github.com/openshift/openshift-docs/pull/62149 will need merged alongside the RN PR

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc]: Thanks, I've moved it.

Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc]: Added, thank you!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc]: Fixed, thank you!

**Pull request merged**